packages/web3-plugin-wallet-rpc/README.md

```diff
@@ -1,3 +1,3 @@
 # Web3.js Plugin for Wallet RPC methods
 
-Refer to the [README](../../README.md) file located in the root of this repository.
+Refer to the [README](https://github.com/web3/web3-plugin-wallet-rpc/blob/main/README.md) file located in the root of this repository.
```

Comment:
@danforbes this file is also used on npm as a readme, so maybe this one should be the default one, and the root repo readme a symlink? I dunno.

see https://www.npmjs.com/package/web3-plugin-wallet-rpc

Reply:
Opened https://github.com/web3/web3-plugin-wallet-rpc/pull/25